src/fileio.cpp

```diff
@@ -16,17 +16,18 @@
 #include "fileio.h"
 #include "rev.h"
 #ifdef LINUX
-	#include "unix/fileio_unix.h"
 	#include <dirent.h>
 	#include <errno.h>
 	#include <unistd.h>
 #elif WINDOWS
-	#include "windows/fileio_windows.h"
 	#include <direct.h> // contains chdir in windows
 #endif
 #include <sys/types.h>
-#include <sys/stat.h>
-
+#ifdef LINUX
+const char *DIR_SEP = "/";  ///< Directory separator character.
+#elif WINDOWS
+const char *DIR_SEP = "\\";  ///< Directory separator character.
```

Comment:
Doesn't Windows "tolerate" using a forward slash as directory separator?

Reply:
No idea, it was done like that in the previous implementation. I lack a windows machine so I can't actualliy check that...

Reply:
I found an even better option: `std::filesystem::path::preferred_separator`.

Reply:
This article has good information about the subject in general: https://en.cppreference.com/w/cpp/filesystem/path

Reply:
Sounds good, I'll play with it later on and push the changes when I'm done.